environment.yml

```diff
@@ -5,13 +5,14 @@ channels:
 dependencies:
   - python=3.7
   - pip
-  - pytorch=1.1
-  - torchvision
-  - opencv
   - matplotlib
   - jupyterlab
+  - pytorch=1.4
+  - torchvision=0.5
+  - torchtext
+  - pillow
```

Comment:
I don't think it's necessary. Is it?

Reply:
Yeah, it's not (explicitly) required. I will remove it.